tsptw_data_dt.h

```diff
@@ -468,6 +470,8 @@ class TSPTWDataDT {
     int64 problem_matrix_index;
     int64 value_matrix_index;
     std::vector<int64> vehicle_indices;
+    std::vector<int64> initial_capacities;
+    std::vector<int64> initial_loads;
```

Comment:
We can do `initial_capacity` and `initial_load` because we know that load and capacity info of a vehicle are multidimensional (by units) this is what we do for capacity or overload_multiplier for example.